pkg/backup/backup/backup_manager.go

```diff
@@ -983,6 +992,27 @@ func (bm *backupManager) teardownVolumeBackup(backup *v1alpha1.Backup) (err erro
 	return nil
 }
 
+func (bm *backupManager) deleteVolumeBackupInitializeJob(backup *v1alpha1.Backup) error {
+	ns := backup.GetNamespace()
+	name := backup.GetName()
+	backupInitializeJobName := backup.GetVolumeBackupInitializeJobName()
+
+	backupInitializeJob, err := bm.deps.JobLister.Jobs(ns).Get(backupInitializeJobName)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return nil
```

Comment:
add a warning level log here

Reply:
Job not found is what we expect. It indicates the job already deleted, so I think we don't need warning log here.

Reply:
The init job not found is what we expect, so I add a info log here.